var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -142,6 +151,9 @@ def configure(self, spec, prefix):
         # Compile tools into flag for configure.
         flags.append(f"--tools={','.join(tools)}")
 
+        # Ensure configure prints full versions of variables for better debugging
+        flags.append("--enable-verbose-configure")
```

Comment:
I think it's just `--verbose-configure`. Although I like `enable` better since that'd make more sense lol.

Reply:
I made the mistake of looking at the flags on main rather than the feature tag, seems it changed. Either way, pulled it out for now until we can at least get something that builds correctly. I'm a bit shocked how funky this config setup has to be honestly, the original should have been fine, and even reading the code for cargo I couldn't see any reason it should have been truncated. Really strange.

Reply:
Turns out same for http.cainfo. It didn't exist when 1.70 was released in June or July. Have to find a cross-version option or update or both. Will look at this again in a bit.

Reply:
It's back to using the env var, must have been something else, but _IT BUILDS_.  Working on adding the three newer rust versions so we can test that it still works.  Have a short list of other packages I'd like to get in here, but right now, want to get to building and testing regularly on linux, then maybe mac after, first.

Reply:
Aaaad it breaks radiuss on ancient ubuntu. Gosh darn it. 🤦 https://gitlab.spack.io/spack/spack/-/jobs/9076757